e2e/__tests__/__snapshots__/testTodo.test.ts.snap

```diff
@@ -45,12 +45,20 @@ FAIL __tests__/todoNoArgs.test.js
       at Object.todo (__tests__/todoNoArgs.test.js:8:4)
 `;
 
+exports[`shows todo messages when in verbose mode 1`] = `
+PASS __tests__/verbose.test.js
+  todos in verbose reporter
+    ✎ todo this
+    ✎ todo should
+    ✎ todo work
+`;
+
 exports[`works with all statuses 1`] = `
 FAIL __tests__/statuses.test.js
   ✓ passes
   ✕ fails
   ○ skipped 1 test
-  ✎ todo 1 test
+  ✎ todo todo
```

Comment:
should we put it above `skipped`?

And should `skipped` also list out all instead of summarizing?

Reply:
If we're not printing out all of the skipped tests, let's leave it below so it'll look neater having

```
pass
fail
skip
todo
todo
todo
```

But if we're printing skipped as well, we could go either way